src/games/marooned.rs

```diff
@@ -359,17 +361,17 @@ use Status::*;
 /// The game state
 #[derive(Clone, Debug, Default, PartialEq, Eq, Serialize, Deserialize)]
 pub struct GameState {
-    settings: Settings,
-    history: Vec<Action>,
+    settings: Arc<Settings>,
```

Comment:
This seems correct right? I want to clone the game state before "mutating" it. Wrapping `Settings` in an `Arc` makes it very cheap to clone